src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -27,6 +27,19 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
     );
   }
 
+  if (type === 'ANON_TOPIC') {
+    return (
+      <View
+        style={[styles.containerImage, styles.containerDefaultImage, styles.containeranon_primary]}>
+        <FastImage
+          source={CommunityIcon}
+          resizeMode={FastImage.resizeMode.contain}
+          style={styles.imageDefaultCommunity}
+        />
+      </View>
+    );
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new condition has been added to handle the case when `type` is `'ANON_TOPIC'`. The logic seems correct as it returns a styled view with an image for this type. However, consider extracting the repeated JSX into a separate component or function to avoid duplication.
```diff
+ if (type === 'ANON_TOPIC') {
+   return (
+     <View
+       style={[styles.containerImage, styles.containerDefaultImage, styles.containeranon_primary]}>
+       <FastImage
+         source={CommunityIcon}
+         resizeMode={FastImage.resizeMode.contain}
+         style={styles.imageDefaultCommunity}
+       />
+     </View>
+   );
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->